### PR TITLE
added: wishlist page

### PR DESCRIPTION
**Pull request by rkapoor10**
wishlist

### CODE REVIEW

File: wishlist/wishlist.html [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+                    <p class="price-box txt-xs fw-bold">Rs. 1345 <s class="striked">Rs. 1495</s> <span class="discount darkorange">(10% OFF)</span></p>
+                    <button class="btn-secondary txt-xs">ADD TO CART</button>
+                </div>
+
+                <div class="card-container">
+                    <div class="parent-image">
+                        <img class="image-container" src="https://picsum.photos/300/400" alt="">
+                        <div class="tags-flex">
+                            <button class="btn-round btn-heart txt-s lightteal child-wishlist"><i class="fas fs-15 fa-times-circle"></i></button>
+                        </div>
```
Comment (JantuDeb) [inline wishlist/wishlist.html:97]: You can remove the parent `div`

**Review by JantuDeb — APPROVED** [review]
Suggested a changes

File: wishlist/wishlist.html [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+
+<body>
+    <div class="wishlist-pg-layout">
+        <!-- NAVBAR SECTION------------------------------------------------------------------------------ -->
+        <div class="child wishlist-pg-navbar">
+            <header class="navbar-container-flex bg-white">
+                
+                <div class="nav-title">
+                    <div class="nav-logo">
+                        <img src="/assets/rallyalley-logo.png" alt="logo" srcset="" />
```
Comment (harshitadaswani) [inline wishlist/wishlist.html:23]: remove `srcset` if not in use

**Review by harshitadaswani — APPROVED** [review]
Please refactor mentioned change and merge the PR

File: wishlist/wishlist.html (hunk 1 above)
Comment (rkapoor10) [reply wishlist/wishlist.html:97]: ok

File: wishlist/wishlist.html (hunk 2 above)
Comment (rkapoor10) [reply wishlist/wishlist.html:23]: done

**Pull request merged**